typeclasses/characters.py

```diff
@@ -84,6 +86,101 @@ def at_pre_unpuppet(self):
         if self.nattributes.has('combat_handler'):
             self.ndb.combat_handler.remove_character(self)
 
+    def return_appearance(self, looker):
+        """
+        This formats a description. It is the hook a 'look' command
+        should call.
+        Args:
+            looker (Object): Object doing the looking.
+        """
+        if not looker:
+            return ""
+
+        # get description, build string
+        name = self.get_display_name(looker)
+        per  = self.traits.PER
+
+        # These each do a skill check, but they always pass
+        # if you're looking at yourself
+        knows_race = skill_check(per) or looker == self
+        knows_archetype = skill_check(per) or looker == self
+        knows_health = skill_check(per) or looker == self
+        knows_stamina = skill_check(per) or looker == self
```

Comment:
I think that's an excellent idea - I actually asked about that on the issue related to this PR.

I think race should be automatic - race will be fairly easy to tell.
Archetype would be harder - since the skills that exist are relatively simple, maybe setting a higher threshold, maybe 7?

Any suggestions on the health? My thought was to make it based on medicine, not perception.